<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>240</width><height>320</height></rect>
  </property>
  <property name="windowTitle"><string>Calculadora</string></property>

  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">

    <item>
     <widget class="QLineEdit" name="lineEdit_Display">
      <property name="readOnly"><bool>true</bool></property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="minimumHeight"><number>40</number></property>
      <property name="font">
       <font><pointsize>14</pointsize></font>
      </property>
     </widget>
    </item>

     <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <!-- 1ª linha -->
       <item row="0" column="0"><widget class="QPushButton" name="btn7"><property name="text"><string>7</string></property></widget></item>
       <item row="0" column="1"><widget class="QPushButton" name="btn8"><property name="text"><string>8</string></property></widget></item>
       <item row="0" column="2"><widget class="QPushButton" name="btn9"><property name="text"><string>9</string></property></widget></item>
       <item row="0" column="3"><widget class="QPushButton" name="btnDiv"><property name="text"><string>/</string></property></widget></item>

       <!-- 2ª linha -->
       <item row="1" column="0"><widget class="QPushButton" name="btn4"><property name="text"><string>4</string></property></widget></item>
       <item row="1" column="1"><widget class="QPushButton" name="btn5"><property name="text"><string>5</string></property></widget></item>
       <item row="1" column="2"><widget class="QPushButton" name="btn6"><property name="text"><string>6</string></property></widget></item>
       <item row="1" column="3"><widget class="QPushButton" name="btnMul"><property name="text"><string>*</string></property></widget></item>

       <!-- 3ª linha -->
       <item row="2" column="0"><widget class="QPushButton" name="btn1"><property name="text"><string>1</string></property></widget></item>
       <item row="2" column="1"><widget class="QPushButton" name="btn2"><property name="text"><string>2</string></property></widget></item>
       <item row="2" column="2"><widget class="QPushButton" name="btn3"><property name="text"><string>3</string></property></widget></item>
       <item row="2" column="3"><widget class="QPushButton" name="btnSub"><property name="text"><string>-</string></property></widget></item>

       <!-- 4ª linha -->
       <item row="3" column="0" colspan="2"><widget class="QPushButton" name="btn0"><property name="text"><string>0</string></property></widget></item>
       <item row="3" column="2"><widget class="QPushButton" name="btnClear"><property name="text"><string>C</string></property></widget></item>
       <item row="3" column="3"><widget class="QPushButton" name="btnAdd"><property name="text"><string>+</string></property></widget></item>

       <!-- 5ª linha (botão "=" ocupa a linha toda) -->
       <item row="4" column="0" colspan="4"><widget class="QPushButton" name="btnEqual"><property name="text"><string>=</string></property></widget></item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>

  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
